nbi/buildspec.yml

```diff
@@ -20,12 +20,13 @@ phases:
       - echo "Setting up conda"
       - export PATH=$LCC_CONDA_INSTALL_DIR/bin:$PATH
       - for CHANNEL in conda-forge cf-staging; do conda config --system --append channels $CHANNEL; done
+      - conda install --force-reinstall --yes conda-pack setuptools pip wheel
```

Comment:
What if we specify the versions we require, may be add that before conda-pack as well? 
Also, might be worth looking into this: https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/pip-interoperability.html

It might slow down conda, but let's take a look by what factor.

Reply:
Attempted the combination of these and each uniquely  but still ran into the same issue.